GoodDollar/GoodDAPP
# (Bug) 854 payment request should show survey only after confirm and show correct from

# Description
- survey shown only after confirmation
 - confirmation screen shows correct from.
 - if payment request address=current wallet address then ignore and dont start the payment request process

About #854 

# How Has This Been Tested?
u need two accounts. Also, u need to test two cases:
1. create receive code for one account and use it in another
2. create receive code and use it with same account
check definition of done

# Checklist:
